augur/refine.py

```diff
@@ -242,7 +242,10 @@ def run(args):
             elif args.root in ['least-squares', 'min_dev', 'oldest']:
                 raise TypeError("The rooting option '%s' is only available when inferring a timetree. Please specify an explicit outgroup."%args.root)
             else:
-                T.root_with_outgroup(args.root)
+                try:
+                    T.root_with_outgroup(args.root)
+                except ValueError as err:
+                    raise ValueError(f"{err}\nHINT: This error may be because your specified root with name '{args.root}' was not found in your alignment file")
```

Comment:
We can use exception chaining here to get the `The above exception was the direct cause of the following exception:` message in the tracebacks and not duplicate the `err` message.
```python3
except ValueError as err:
        raise ValueError("Custom error message...") from err
```

Reply:
Sounds good!

Reply:
Should be resolved now